diff --git a/Makefile b/Makefile
index c52b50b..5e7f2ec 100644
--- a/Makefile
+++ b/Makefile
@@ -289,6 +289,17 @@ else ifeq ($(platform), rpi1)
 	MMAP_JIT_CACHE = 1
 	HAVE_DYNAREC = 1
 
+# @DEVICE@
+else ifeq ($(platform), @DEVICE@)
+     CPUFLAGS :=  -march=armv8-a -mtune=@TARGET_CPU@ -mabi=lp64 -Wno-psabi
+     CFLAGS  := $(CPUFLAGS) -fomit-frame-pointer -ffast-math
+     TARGET := $(TARGET_NAME)_libretro.so
+     fpic := -fPIC
+     SHARED := -shared -Wl,--version-script=link.T -Wl,--no-undefined
+     CPU_ARCH := arm64
+     MMAP_JIT_CACHE = 1
+     HAVE_DYNAREC = 1
+
 # Classic Platforms ####################
 # Platform affix = classic_<ISA>_<µARCH>
 # Help at https://modmyclassic.com/comp
